src/useMobxEffect.ts

```diff
@@ -0,0 +1,33 @@
+import { useMemo, useRef } from "react"
+import { useUnmount } from "./utils"
+
+type Disposer = () => any
```

Comment:
Why not to use `IReactionDisposer` from mobx directly?

Reply:
good point :)

Reply:
Well, if someone wants to do non-mobx stuff, then they can use `useEffect`. Imo no point in being super universal really :)

Reply:
yeah, well, the funny thing will be that it will work in JS and it won't work in TS, but I'll change it